### PR TITLE
Added tabs to header

### PR DESCRIPTION
**Pull request by slaveeks**
1. Added Tab component
2. Configured `postcss-nested`
3. Added `codex-icons`
4. Added tabs to Header

For now home page looks like this: 
image

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
-    <Header />
+    <Header :tabs="tabs"/>
     <router-view></router-view>
   </div>
 </template>
 
 <script lang="ts" setup>
 import Header from './presentation/components/Header.vue';
+import {IconPicture} from "@codexteam/icons";
+const tabs = [
```
Comment (neSpecc) [inline src/App.vue:11]: I'd suggest moving this to the Header component

File: src/presentation/components/Header.vue [hunk 2]
```diff
@@ -1,18 +1,67 @@
 <template>
   <div class="header">
+    <div class="header__tabs">
+      <Tab v-for="tab in tabs" :title="tab.title" :path="tab.path" :icon="tab.icon" :isDefault="tab.isDefault" :isActive="tab.isActive" :onClose="tab.onClose" :key="tab.path"/>
```
Comment (neSpecc) [inline src/presentation/components/Header.vue:4]: move each attribute to a separate line, and connect eslint plugin `plugin:vue/vue3-recommended` please

File: src/presentation/components/ui/Tab.vue [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    overflow: hidden;
+    text-overflow: ellipsis;
+    line-clamp: 1;
+    white-space: nowrap;
+  }
+
+  &__close {
+    height: var(--size-icon);
+    width: var(--size-icon);
+    align-items: center;
```
Comment (neSpecc) [inline src/presentation/components/ui/Tab.vue:81]: seems redundant

File: src/presentation/components/ui/Tab.vue [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  text-decoration: none;
+  height: var(--header-tab-height);
+  max-width: var(--header-tab-max-width);
+  background: var(--bg-color-tab);
+  border-radius: var(--radius-m);
+  padding: var(--spacing-xxs) var(--spacing-ms);
+  color: var(--text-color-header);
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
```
Comment (neSpecc) [inline src/presentation/components/ui/Tab.vue:61]: flex start?

File: src/presentation/styles/colors.pcss [hunk 5]
```diff
@@ -1,3 +1,6 @@
 #app {
   --bg-color-header: #1A1B1C;
+  --text-color-header: #979BA0;
+  --text-color-header-active: #fff;
```
Comment (neSpecc) [inline src/presentation/styles/colors.pcss:4]: seems redundant, get variable names from figma

File: src/presentation/styles/sizes.pcss [hunk 6]
```diff
@@ -17,4 +17,7 @@
   --layout-content-width: 700px;
 
   --layout-header-height: 52px;
+
+  --header-tab-max-width: 138px;
+  --header-tab-height: 32px;
```
Comment (neSpecc) [inline src/presentation/styles/sizes.pcss:22]: use --layout prefix for all sizes

Comment (neSpecc) [inline src/presentation/styles/sizes.pcss:22]: --header-tab-height: 32px;

do we really need this?

File: src/presentation/components/Header.vue (hunk 2 above)
Comment (kseniaSs) [reply src/presentation/components/Header.vue:4]: @neSpecc Can we add prettier too? Or it's not neccessary?

File: src/App.vue (hunk 1 above)
Comment (slaveeks) [reply src/App.vue]: done

File: src/presentation/components/ui/Tab.vue (hunk 3 above)
Comment (slaveeks) [reply src/presentation/components/ui/Tab.vue]: removed

File: src/presentation/components/ui/Tab.vue (hunk 4 above)
Comment (slaveeks) [reply src/presentation/components/ui/Tab.vue]: changed

File: src/presentation/styles/colors.pcss (hunk 5 above)
Comment (slaveeks) [reply src/presentation/styles/colors.pcss]: done

File: src/presentation/styles/sizes.pcss (hunk 6 above)
Comment (slaveeks) [reply src/presentation/styles/sizes.pcss]: removed

File: src/presentation/components/Header.vue [hunk 7]
```diff
@@ -1,18 +1,46 @@
...
   </div>
 </template>
 
 <script lang="ts" setup>
+import Tab from './ui/Tab.vue';
+import { IconPicture } from '@codexteam/icons';
+
+const tabs = [
+  {
+    title: 'Home',
```
Comment (neSpecc) [inline src/presentation/components/Header.vue:24]: use i18n here

File: src/presentation/components/ui/Tab.vue [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+  /**
+   * Link path
+   */
+  path: string;
+
+  /**
+   * Is tab default, in other words, can't be closed
+   */
+  isDefault?: boolean;
```
Comment (neSpecc) [inline src/presentation/components/ui/Tab.vue:51]: maybe `isPinned`?

File: src/presentation/components/ui/Tab.vue [hunk 9]
```diff
@@ -0,0 +1,107 @@
+<template>
+  <router-link
+    :to="path"
+    class="tab"
+    :class="{'tab--active': isActive}"
+  >
+    <div
```
Comment (neSpecc) [inline src/presentation/components/ui/Tab.vue:7]: we need to find a way of using icons via a component, like:

```vue
<Icon name="IconCross"/>
```
orr 

```vue
<IconCross />
```

File: src/presentation/components/Header.vue (hunk 2 above)
Comment (neSpecc) [reply src/presentation/components/Header.vue]: > @neSpecc Can we add prettier too? Or it's not neccessary?

Let's try

Comment (neSpecc) [reply src/presentation/components/Header.vue]: > > @neSpecc Can we add prettier too? Or it's not neccessary?
> 
> Let's try

Better in a separate PR i guess

File: src/presentation/components/Header.vue (hunk 7 above)
Comment (slaveeks) [reply src/presentation/components/Header.vue]: done

File: src/presentation/components/ui/Tab.vue (hunk 8 above)
Comment (slaveeks) [reply src/presentation/components/ui/Tab.vue]: renamed

**Review by neSpecc — APPROVED** [review]

File: src/presentation/styles/colors.pcss [hunk 10]
```diff
@@ -1,3 +1,6 @@
 #app {
   --bg-color-header: #1A1B1C;
+  --color-text-main: #94979A;
```
Comment (kseniaSs) [inline src/presentation/styles/colors.pcss:3]: --bg-color-tab && --color-text-main seem to be different different. So maybe --text-color-main or --text-main-color?

Comment (neSpecc) [reply src/presentation/styles/colors.pcss:3]: Fixed in #19

File: src/presentation/components/ui/Tab.vue [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+      @click="onClose"
+      v-html="IconCross"
+    />
+  </router-link>
+</template>
+
+<script setup lang="ts">
+import { IconCross } from '@codexteam/icons';
+
+interface TabProps {
```
Comment (kseniaSs) [inline src/presentation/components/ui/Tab.vue:27]: We separate this, don't we?

Comment (slaveeks) [reply src/presentation/components/ui/Tab.vue]: moved to file `TabProps.ts`

**Pull request merged**